Gems/AudioSystem/Code/Source/Engine/FileCacheManager.cpp

```diff
@@ -830,6 +830,11 @@ namespace Audio
                 }
                 else
                 {
+                    if (!audioFileEntry->m_asyncStreamRequest)
+                    {
+                        audioFileEntry->m_asyncStreamRequest = streamer->CreateRequest();
+                    }
+
                     streamer->Read(
```

Comment:
Is the `audioFileEntry->m_asyncStreamRequest` always coming here null, or "sometimes"?

If the request is always null here, maybe this should be changed to:
```
audioFileEntry->m_asyncStreamRequest = streamer->Read(
    audioFileEntry->m_filePath.c_str(),
    ...
```